volatility3/framework/symbols/windows/extensions/registry.py

```diff
@@ -132,6 +175,103 @@ def get_full_key_name(self) -> str:
         return "\\".join(reversed(output))
 
 
+class CM_KEY_VALUE(objects.StructType):
```

Comment:
It;s really hard to see what's changed when this has (mostly) just moved position in the file).  I realize it's been done to allow for typing, but I'd almost sooner split it into two PRs, one where the type is in quotes for delayed resolution, and then another shifting the code about...  5:S

Reply:
Moved it back to it's original position 👍